src/api/rest_api.ts

```diff
@@ -50,6 +51,22 @@ function backend_user_to_frontend_user(user: any): User {
   };
 }
 
+
+function backend_user_profile_to_frontend_user_profile(user: any): UserProfile {
+  return {
+    uuid: user.uuid,
```

Comment:
to przepakowanie chyba nie ma sensu? W innych miejscach to robiłem bo to co zwracał backend rozjeżdżało się z tym co miałem zrobione na froncie.